core/mgmt/v1beta/metric.proto

```diff
@@ -29,26 +29,13 @@ enum Status {
 
 // ========== Pipeline endpoints
 
-// PipelineTriggerRecord represents a pipeline execution event.
-message PipelineTriggerRecord {
```

Comment:
just to make sure, these are not retired but will be moved to under pipeline/model as logging feature right?

Reply:
Exactly, the trigger list will be powered by the logging feature. However, it is a good opportunity to confirm that this endpoint (trigger list) doesn't have any clients, right?